definition-src/Game/LambdaHack/Definition/DefsInternal.hs

```diff
@@ -58,7 +58,7 @@ displayContentSymbol = id
 -- by accident (this is still possible via conversion functions,
 -- if one insists, so the abstraction is leaky, but that's fine).
 newtype ContentSymbol c = ContentSymbol Char
-  deriving (Show, Eq, Ord)
+  deriving (Show, Eq, Ord, Generic, Binary, NFData)
```

Comment:
Coudl you remove the old code above, starting with TODO up to and including `{-` and then the single line with `-}` below. I'm actually baffled that this works. Aren't there errors or warnings that `ContentSymbol` has a duplicated definition, etc.? Or is github showing a broken diff?